.travis.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         - make .gitvalidation
         - make verify
         - make binaries
-      go: 1.8.x
+      go: 1.9.x
```

Comment:
we could build both for a while..

Reply:
Done.

Reply:
@mikebrow Sorry, just saw your comment.

Actually both containerd and Kubernetes have moved to 1.9 already. I think we should build with 1.9. :)

Reply:
Ok fair enough, time to move up then... waves good bye to 1.8